sqe-http-api/Services/UserService.cs

```diff
@@ -104,16 +122,110 @@ public UserDTO GetCurrentUser()
             return null;
         }
 
+        /// <summary>
+        /// This will create a new user account in the database and email an authorization token to the user.
+        /// </summary>
+        /// <param name="newUserData">All of the information for the new user.</param>
+        /// <returns>Returns a UserDTO for the newly created user account.</returns>
+        public async Task<UserDTO> CreateNewUserAsync(NewUserDTO newUserData)
+        {
+            // Ask the repo to create the new user
+            var createdUser = await _userRepository.CreateNewUserAsync(newUserData.userName, newUserData.email, newUserData.password,
+                newUserData.forename, newUserData.surname, newUserData.organization);
+            
+            // Email the user
+            // TODO: Add link to web endpoint when we know what that is. Can token be in URL query?
+            const string emailBody = @"
```

Comment:
This should probably go into another file, a resource maybe.

Reply:
Yeah, I like the idea you mentioned above to use a Razor template, so I am going to push this off for now till we have that set up.